pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -77,6 +77,11 @@ func (ctrl *Controller) syncNodeConfigHandler(key string) error {
 		err := fmt.Errorf("could not fetch Node: %w", err)
 		return err
 	}
+	// Set the Cgroups version explicitly to "v1"
+	if nodeConfig.Spec.CgroupMode == emptyInput {
```

Comment:
can this be nil somehow? I forget how `omitempty` works, or does this check cover that already

Reply:
`nodeConfig` can't be nil here as we return error [above](https://github.com/openshift/machine-config-operator/blob/c916a0fd9f410397741e9248c2e1038371872a06/pkg/controller/kubelet-config/kubelet_config_nodes.go#L78) if it is `nil`